src/security/cert_generators/file/certificate_db.h

```diff
@@ -14,6 +14,9 @@
 #if HAVE_OPENSSL_OPENSSLV_H
 #include <openssl/opensslv.h>
 #endif
+#if HAVE_OPENSSL_LHASH_H
+#include <openssl/lhash.h>
```

Comment:
This addition puzzles me. Either it is not needed because `compat/openssl.h` includes it OR this file is used where `compat/openssl.h` itself is _not_ included.

Reply:
I think this was left over from before the shim was used.  I'll remove it.
BTW, `<openssl/opensslv.h>` could also be dropped here, as all it does is define `(OPENSSL|SHLIB)_VERSION_.*` macros, which are not being used anywhere this file is included.